### ISSUE TITLE
An Incompatability issue

### ISSUE THREAD

**Issue by ZoenkBoenk** [fzzyhmstrs/pc#1]
I just downloaded the mod and I have around 30 other mods installed such as sodium which most of them are also performance mods.

If needed then here is the crash log.

A mod crashed on startup!
net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'advanced_runtime_resource_pack'!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:162)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'advanced_runtime_resource_pack'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:160)
	... 2 more
Caused by: net.fabricmc.loader.api.EntrypointException: Exception while loading entries for entrypoint 'preLaunch' provided by 'advanced_runtime_resource_pack'
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:56)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
	... 3 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.devtech.arrp.ARRP failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:467)
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50)
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117)
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53)
	... 4 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [xlpackets.mixins.json:PacketByteBufMixin from mod xxlpackets] from phase [DEFAULT] in config [xlpackets.mixins.json] FAILED during PREPARE
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinPrepareError(MixinProcessor.java:585)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:543)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:462)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:438)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:290)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 13 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTargetAlreadyLoadedException: Critical problem: xlpackets.mixins.json:PacketByteBufMixin from mod xxlpackets target net.minecraft.class_2540 was loaded too early.
	at org.spongepowered.asm.mixin.transformer.MixinInfo.readDeclaredTargets(MixinInfo.java:948)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:882)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:852)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:777)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:540)
	... 19 more

**Comment by ZoenkBoenk** [comment]
Btw I do not have a mod called advanced runtime resource pack

**Comment by fzzyhmstrs** [comment]
ARRP is probably included inside of another mod. The problem seems to be with "XXL Packets". Let me try to replicate the issue from my side and see if there's any way to fix from my end.

**Comment by zndrmn** [comment]
I can try and help with debugging (finding the mod), having the same issue

**Comment by fzzyhmstrs** [comment]
version 0.1.2 resolves this issue (at least with XXL Packets, @zndrmn if you still have the issue with 0.1.2 please open a new ticket with your error)

**Comment by ZoenkBoenk** [comment]
After posting this I tried to find out what mod caused this as I saw this before and I found out that XXLpackets was the mod, then I just forgot abt this. THO it was worth it bc I have been searching for a mod like this for a year by now and even if this is recent I'm really happy that you made this mod. We all thank you!!! Also ty for fixing the bug now I can use XXLpackets again. (Also went from 180 to 300 w 32 render distance and mods like nvidium) I legit have every performance mode that is possible to be downloaded for MC 1.20.4
